web/geo/validate-v2.js

```diff
@@ -0,0 +1,187 @@
+const {
+  area,
+  bbox,
+  bboxPolygon,
+  convex,
+  dissolve,
+  featureCollection,
+  featureEach,
+  getGeom,
+  intersect,
+} = require('@turf/turf');
+const { getPolygons, getHoles, writeJSON, log } = require('./utilities');
+
+const { getZonesJson } = require('./files');
+
+// TODO: Improve this function so each check returns error messages,
+// so we can show all errors instead of taking them one at a time.
+function validateGeometryV2(fc, config) {
+  const fcHighFidelity = {
+    ...fc,
+    features: fc.features.filter((feature) => {
+      try {
+        return !feature.properties.aggregatedView;
+      } catch (e) {
+        console.log('Error: ', e, 'Feature: ', feature); // eslint-disable-line no-console
```

Comment:
Hmm, I am wondering why this try/catch is required?

Reply:
It's not required but it's useful for debugging if the input data has any issues